pipelines/upgrade/05-server_to_intermediate.yml

```diff
@@ -19,6 +19,8 @@
       scenario: "{{ pipeline_type }}"
       scenario_os: "{{ pipeline_os }}"
       scenario_version: "{{ forklift_upgrade_version_intermediate }}"
+    - role: update_tomcat
+      when: pipeline_type == 'katello'
```

Comment:
and pipeline_os centos8-stream? Otherwise this will get executed on Alma and fail?

Reply:
Good point, done.